include/psp2/touch.h

```diff
@@ -18,9 +18,9 @@ extern "C" {
 
 #include <psp2/types.h>
 
-// SCE_TOUCH_MAX_REPORT = 8
 enum {
-	SCE_TOUCH_MAX_REPORT	= 8	//!< FIXME 6 on front | 4 on back
```

Comment:
I know but it's just wrong. Not a big deal to just change the variable name in homebrews that use it. Even this got merged and broke tons of homebrews cause sceRtcTime became sceDateTime: https://github.com/vitasdk/vita-headers/commit/2b10387cc0db941a216097dcb631f6e85607b95f